src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-accessible_output2.py

```diff
@@ -1,19 +1,8 @@
-# ------------------------------------------------------------------
-# Copyright (c) 2020 PyInstaller Development Team.
-#
-# This file is distributed under the terms of the GNU General Public
-# License (version 2.0 or later).
-#
-# The full license is available in LICENSE.GPL.txt, distributed with
-# this software.
-#
-# SPDX-License-Identifier: GPL-2.0-or-later
-# ------------------------------------------------------------------
-
 """
 accessible_output2: http://hg.q-continuum.net/accessible_output2
 """
 
 from PyInstaller.utils.hooks import collect_dynamic_libs
 
-binaries = collect_dynamic_libs('accessible_output2')
+binaries = []
+datas = collect_dynamic_libs('accessible_output2')
```

Comment:
This generally doesn't make any difference. What goes wrong if PyInstaller thinks of those binaries as binaries?